Koronavilkku/Koronavilkku/SceneDelegate.swift

```diff
@@ -65,6 +65,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         activationTask = ExposureManagerProvider.shared.activated.sink { activated in
             Environment.default.exposureRepository.refreshStatus()
         }
+        
+        LocalStore.shared.removeExpiredExposures()
```

Comment:
Could we use earlier lifecycle method to clear expired exposures, see sceneWillEnterForeground